src/VisualStudio/Core/Def/Implementation/Preview/TopLevelChange.cs

```diff
@@ -129,6 +131,8 @@ private Solution ApplyFileChanges(Solution solution, IEnumerable<FileChange> fil
                     {
                         solution = isAdditionalDoc ?
                             solution.RemoveAdditionalDocument(updatedDocumentIdOpt) :
+                            isAnalyzerConfigDoc ?
+                            solution.RemoveAnalyzerConfigDocument(updatedDocumentIdOpt) :
                             solution.RemoveDocument(updatedDocumentIdOpt);
```

Comment:
nit: switch to `?:` on subsequent lines to better match IDE coding patterns (and make this easier to read :)).

nit: these sorts of fall-through patterns concern me (As per previous comments).